data/aashish05.json

```diff
@@ -0,0 +1,13 @@
+{
+"name": "Aashish Kumar",
+"irc": "virus",
+"irc_channels": [rust],
```

Comment:
This field needs to be written as a JSON array of JSON strings, in this case `["rust"]` rather than `[rust]`.